buildenv/jenkins/getDependency

```diff
@@ -0,0 +1,38 @@
+#!groovy
+
+LABEL='ci.role.test&&hw.arch.x86&&sw.os.linux'
+
+stage('Queue') {
+    node("$LABEL") {
+       cleanWs()
+       testBuild()
+    }
+}
+
+def testBuild() {
+	def time_limit = 8
+	if(params.TIME_LIMIT) {
+		time_limit = params.TIME_LIMIT.toInteger()
+	}
+	timeout(time: time_limit, unit: 'HOURS') {
+		try {
+			if( params.BUILD_TYPE == "systemtest" ){
+			    sh 'curl -OLJks  "https://api.adoptopenjdk.net/v2/binary/nightly/openjdk8?openjdk_impl=hotspot&os=linux&arch=x64&release=latest&type=jdk"'
+                sh 'mkdir ${WORKSPACE}/j2sdk-image'
+                sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
+                sh '${WORKSPACE}/j2sdk-image/jre/bin/java -version'
+                sh 'git clone git://github.com/AdoptOpenJDK/openjdk-systemtest'
+                sh 'ant -f ./openjdk-systemtest/openjdk.test.mauve/build.xml -Djava.home=${WORKSPACE}/j2sdk-image/jre -Dprereqs_root=${WORKSPACE}/systemtest_prereqs'
```

Comment:
What's the requirement of `java.home` to build mauve.jar? If it's not necessarily the test against JAVA grabbing a stable JAVA( release not a nightly) may be better.

Reply:
java.home is required by the systemtest scripts. Yes. I agree. Can you share link to a stable sdk tar file?